tests/test_parent_domain.py

```diff
@@ -8,6 +8,7 @@
 from pyverbs.srq import SrqAttr, SrqInitAttr, SRQ
 from pyverbs.qp import QPInitAttr, QP
 from tests.base import BaseResources
+from tests.base import RDMATestCase
 import pyverbs.mem_alloc as mem
 import pyverbs.device as d
```

Comment:
Please remove this import "import pyverbs.device as d", since it's no longer used after your last changes.

Reply:
I'll push a fix soon.